SerialPrograms/Source/NintendoSwitch/Programs/NintendoSwitch_GameEntry.cpp

```diff
@@ -116,10 +122,31 @@ void close_game_from_home(ConsoleHandle& console, ProControllerContext& context)
 

```

Comment:
You're actually gonna want to split this function into to, a blind one with the old behavior and the new feedback one. Since some of the older SwSh programs are able to run without feedback.

So give this the same treatment as `start_game_from_home()`. Rename the current one to  `start_game_from_home_blind()`, then add the new feedback one.